Dockerfile

```
@@ -41,36 +41,33 @@ RUN apt-get clean \
   && apt-get update \
   && apt-get install -y \
   build-essential python3.9 python3-pip git cmake make ruby rake autoconf automake \
-  && pip install conan -v "conan==1.59.0"
+  && pip install conan -v "conan==2.0.9"
```

I have no idea about your quote preference, however, I prefer using single quotes when I don’t need to use variables (i.e. when I need/want to quote a static string). OFC, this would need to be changed in the entire file.

```suggestion
  && pip install conan -v 'conan==2.0.9'
```